core-web/libs/portlets/edit-ema/portlet/src/lib/utils/index.ts

```diff
@@ -214,3 +214,14 @@ export function createPageApiUrlWithQueryParams(
 
     return queryParams.length ? `${url}?${queryParams}` : url;
 }
+
+/**
+ * Check if the variant is the default one
+ *
+ * @export
+ * @param {string} [variant]
+ * @return {*}  {boolean}
+ */
+export function getIsDefaultVariant(variant?: string): boolean {
```

Comment:
```suggestion
export function isDefaultVariant(variant?: string): boolean {
```

Reply:
I have a variable called like that, that's why is called `getIsDefaultVariant`